contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -720,6 +720,28 @@ static bool
 pltsql_bbfViewHasInsteadofTrigger(Relation view, CmdType event)
 {
 	TriggerDesc *trigDesc = view->trigdesc;
+	if (triggerOids)
+	{
+		int i;
+		for (i = 0; i < trigDesc->numtriggers; i++)
+		{
+			Trigger *trigger = &trigDesc->triggers[i];
+			Oid current_tgoid = trigger->tgoid;
+			Oid prev_tgoid = InvalidOid;
+			prev_tgoid = lfirst_oid(list_tail(triggerOids));
```

Comment:
Where is the place add oid into triggerOids ?

Reply:
Added in backend code
https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish/pull/245/files
src/backend/commands/trigger.c